andela/denethor-ah-backend
# #163478797 Implement comment feature for articles

#### What does this PR do?
This pull requests adds comment functionality for articles written. Users can comment on articles

#### Description of Task to be completed?
- create controller
- create route
- write tests

#### How should this be manually tested?
- Clone the repo
- Checkout branch `feature/163478764/post-article-comments`
- Run `npm i` to install dependencies
- Set up `.env` file as specified in the `env_sample` file
- Start server with `npm run start:dev`
- Open [Postman](https://www.getpostman.com/)
- Create and verify a new user, then create an article
- Navigate to the `/api/articles/:articleId/comments`
- Supply the ID of the article you created as the `articleId` parameter
- Create a comment

#### What are the relevant pivotal tracker stories?
[#163478764](https://www.pivotaltracker.com/story/show/163478764)

#### Extra info
Also fixed the duplicate houndCI badge and updated the coveralls badge to reflect staging coverage, not master.
